src/utils/common.ts

```diff
@@ -165,7 +165,7 @@ export function isWrappedWith(val: string, start: string, end: string, offset: n
  * @returns Whether the value is quoted or not.
  */
 export function isQuoted(val: string): boolean {
-	if (/^(["'])(.*)\1$/.test(val)) {
+	if (/^(["'`])(.*)\1$/.test(val)) {
```

Comment:
If I remember correctly, backtick quoted strings are pre processed by pug as JS. Therefore it is handled a bit differently than normal quoted strings.
But beside that, if this is the case, I should find a test case to cover that 🤔 
Will have a deeper look later.

Reply:
@bitencode Do you have a case for this? Then add it to the tests
If not, I would like to revert this line and just change it when it is absolutely necessary

Reply:
@Shinigami92 I do not have a specific test case for this. However, with the changes to avoid quoting pug variables assigned to Vue props to other test cases break (`issue-61` - 'should not wrap chained data-bindings' and `issue-80` - 'should not reformat multiline interpolation strings' because they are then not unquoted before being passed to the prettier `format` function.

It's possible that if `isQuoted` need to *not* check for back-tic quotes that there should be another function that does the check needed in `formatDelegatePrettier`.  Or possibly an additional parameter that indicates which quotes to check for with the default being `"'` and the new call can pass in all 3 types of quotes.  I don't know this code base well, or your coding preferences, so I'm open to suggestions.

Reply:
Thx :slightly_smiling_face: I will look into the branch and discover it
Will send feedback later back to you

Reply:
AHA! The thing is, that 61 and 80 have backtick quoted strings, but can be normal strings cause they don't use code interpolation! So no `${` in it.
Then my plugin converts these template literal strings to normal strings (by design)

But I currently don't understand why the new change is needed then :thinking: 

And we should create a test to check that it doesn't convert the backticks if it contains interpolated code in one single line

Reply:
@bitencode So I added commit: https://github.com/prettier/plugin-pug/pull/252/commits/a491e08c6596e2871ced6c5e78e9b61bcfa261d5

Seems that it still works with your change and not work when reverting...
So thx I'm now less afraid and confident using your change :slightly_smiling_face:

Reply:
👍 Awesome - Thanks!